src/strategies/contract-call-multi-outputs/index.ts

```diff
@@ -0,0 +1,40 @@
+import { formatUnits } from '@ethersproject/units';
+import { multicall } from '../../utils';
+
+export const author = 'in19farkt';
+export const version = '0.1.0';
+
+function getArgs(options, address: string) {
+  const args: Array<string | number> = options.args || ['%{address}'];
+  return args.map((arg) =>
+    typeof arg === 'string' ? arg.replace(/%{address}/g, address) : arg
+  );
+}
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+) {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+  const response = await multicall(
+    network,
+    provider,
+    [options.methodABI],
+    addresses.map((address: any) => [
+      options.address,
+      options.methodABI.name,
+      getArgs(options, address)
+    ]),
+    { blockTag }
+  );
+  return Object.fromEntries(
+    response.map((value, i) => [
+      addresses[i],
+      parseFloat(formatUnits(value[options.output].toString(), options.decimals))
```

Comment:
I feel it is better to change `contract-call` strategy itself and make output param optional

```
      parseFloat(formatUnits(options.output ? value[options.output].toString() : value.toString(), options.decimals))
```

Reply:
so i can submit a new pull request which update the contract-call strategy with the code above ?